app/models/version.rb

```diff
@@ -8,11 +8,22 @@ class Version < ApplicationRecord
     class_name: 'Change',
     foreign_key: 'uuid_to'
 
+  after_save :sync_page_title
```

Comment:
I’m a little worried that this will be a lot higher overhead than `after_add` on page association, but it’s probably fine. We could do `after_create`, though.

Reply:
Yeah, it probably will seeing as versions are saved in more contexts than just when they're added to a page. I'm obviously happy to go with whichever you think is best, either `after_save` or `after_create`. For the time being, I'll change it to `after_create`, but let me know if you prefer I leave it be.